### ISSUE TITLE
gallery is not keyboard accessible, needs mouse/click

### ISSUE THREAD

**Issue by r-ferrier** [geeksforsocialchange/gfsc-v3#318]

**Comment by ivan-kocienski-gfsc** [comment]
I would argue this is okay as it is not a popup gallery (which should be keyboard navigable) but is embedded in the page. It would be weird (to me) for an element on the page to react to cursor key presses without any visible indication that it was going to do that. It may also interfere with keyboard navigation.

**Comment by katjam** [comment]
I think it should be possible to access the larger image via keyboard. We may want to look into the best way to announce them, but it seems a shame to not be able to use the gallery without mouse / touch.
